zfsbootmenu/help-files/92/zfsbootmenu.7.ansi

```diff
@@ -136,7 +136,7 @@
 
     [1mzbm.hookroot=<hookspec>[0m
 
-      Tell ZFSBootMenu to attempt to read any early-setup, setup or teardown hooks from the
+      Tell ZFSBootMenu to attempt to read any early-setup, setup, load_key or teardown hooks from the
```

Comment:
You can't build this file by hand. This line is too long. Changes to `zfsbootmenu/help-files` need to be made by `releng/rst2help.sh`.

Reply:
`releng/rst2help.sh` requires `rst2ansi` program. Where to get it?